content/docs/run-api-server/configuring.mdx

```diff
@@ -5,35 +5,111 @@ order: 30
 
-To prepare a database for Horizon's use, you must first ensure the database is blank. It's easiest to simply create a new database on your postgres server specifically for Horizon's use. Next you must install the schema by running `horizon db init`. Remember to use the appropriate command line flags or environment variables to configure Horizon as explained in [Configuring ](./configuring.mdx). This command will log any errors that occur.
+To prepare a database for Horizon's use, you must first ensure the database is blank. It's easiest to simply create a new database on your PostgreSQL server specifically for Horizon's use. Next, you must install the schema by running `horizon db init`. Remember to use the appropriate command line flags or environment variables to configure Horizon as explained [above](#parameters). This command will log any errors that occur.
 
 ### Postgres Configuration
+It is recommended to set `random_page_cost=1` in Postgres configuration if you are using SSD storage. With this setting, Query Planner will make a better use of indices, expecially for `JOIN` queries. We've noticed a huge speed improvement for some queries with this setting.
+
+
+## Configuring Captive Core
+
+While a full Stellar Core node requires a complex configuration with [lots of possible fields](https://github.com/stellar/stellar-core/blob/master/docs/stellar-core_example.cfg), the Captive Core configuration stub can be kept extremely barebones. The rest of the configuration will be generated automagically at runtime. Here's is a minimal working example, operating under the assumption that you want to connect to the testnet and trust SDF's validators exclusively:
+
+<CodeExample>
+
+```toml
+[[HOME_DOMAINS]]
+HOME_DOMAIN="testnet.stellar.org"
+QUALITY="HIGH"
+
+[[VALIDATORS]]
+NAME="sdf_testnet_1"
+HOME_DOMAIN="testnet.stellar.org"
+PUBLIC_KEY="GDKXE2OZMJIPOSLNA6N6F2BVCI3O777I2OOC4BV7VOYUEHYX7RTRYA7Y"
+ADDRESS="core-testnet1.stellar.org"
+HISTORY="curl -sf http://history.stellar.org/prd/core-testnet/core_testnet_001/{0} -o {1}"
+
+[[VALIDATORS]]
+NAME="sdf_testnet_2"
+HOME_DOMAIN="testnet.stellar.org"
+PUBLIC_KEY="GCUCJTIYXSOXKBSNFGNFWW5MUQ54HKRPGJUTQFJ5RQXZXNOLNXYDHRAP"
+ADDRESS="core-testnet2.stellar.org"
+HISTORY="curl -sf http://history.stellar.org/prd/core-testnet/core_testnet_002/{0} -o {1}"
+
+[[VALIDATORS]]
+NAME="sdf_testnet_3"
+HOME_DOMAIN="testnet.stellar.org"
+PUBLIC_KEY="GC2V2EFSXN6SQTWVYA5EPJPBWWIMSD2XQNKUOHGEKB535AQE2I6IXV2Z"
+ADDRESS="core-testnet3.stellar.org"
+HISTORY="curl -sf http://history.stellar.org/prd/core-testnet/core_testnet_003/{0} -o {1}"
+```
+
+</CodeExample>
+
+_(For the remainder of this guide, we'll assume this stub lives at `/etc/default/stellar-captive-core-stub.toml`.)_
+
+
+## Remote Captive Core
+
+If you want to run a Captive Core instance independently of your Horizon instances, the architecture supports that. We provide a package that wraps a Captive Core instance in an HTTP API that Horizon can consume.
```

Comment:
Rather than "independently," it might make sense to say something like "on a separate machine."  Unless, of course, there's some reason to run Captive Core when you're not running Horizon.

Reply:
Done in e7d8d0d9399533c10a83f087d9057eb9f1aaa7c0, I think!

Reply:
Works for me!  Thanks.